src/uwtools/utils/tasks.py

```diff
@@ -4,11 +4,23 @@
 
 import os
 from pathlib import Path
-from shutil import copy
+from shutil import copy, which
+from typing import Union
 
 from iotaa import asset, external, task
 
 
+@external
+def executable(program: Union[Path, str]):
+    """
+    An executable program located on the current path.
+
+    :param program: Name of or path to the program.
+    """
+    yield "Executable program %s" % program
+    yield asset(program, lambda: bool(which(program)))
```

Comment:
I suspect this will be useful later, so I opted to keep it even though it is not used at the moment.